### DISCUSSION TITLE
[A-L] (2023/24) Foglio 9 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#325] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-17 185132](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/dd858563-157f-472b-8bed-4900d1e02e42)

**Comment by daromacarlo** [comment]
![Immagine 2023-11-18 211900](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/01701e9a-d5df-4fc3-97c2-e47859abf23b)
![Immagine 2023-11-18 211837](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/47d63eac-a4e3-4902-86c0-98ad9c32dd74)
![Immagine 2023-11-18 211817](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/f775630f-f2f4-49bc-ba7a-b33877379fbf)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale

![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4271c587-1c0d-4bcc-aef6-c9e774807683)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4195ff69-c6eb-4bb3-bd9d-a7ec61777c88)
